Makefile

```diff
@@ -35,7 +35,7 @@ endif
 
 # keep standard at C11 and C++11
 CFLAGS   = -I.              -O3 -DNDEBUG -std=c11   -fPIC
-CXXFLAGS = -I. -I./examples -O3 -DNDEBUG -std=c++11 -fPIC
+CXXFLAGS = -I. -I./examples -O3 -DNDEBUG -std=c++20 -fPIC
```

Comment:
The last attempt to modernize the C++ standard was reverted (#355). I do agree that non-owning containers are nice, but it seems that the project wants to support old/underdeveloped compilers.

Reply:
But we are using C++ for internals, like examples, and expose only C API for both ggml and llama...

Reply:
what @Piezoid said